workspace/utilities/com/site-nav.xsl

```diff
@@ -85,11 +85,6 @@
 			<xsl:with-param name="attr" select="$attr" />
 			<xsl:with-param name="content">
 
-				<!-- nav-link (COMP: nav-link) -->
-				<xsl:call-template name="nav-link">
-					<xsl:with-param name="attr" select="$nav-link-attr"/>
-					<xsl:with-param name="selected-attr" select="$nav-link-selected-attr" />
-				</xsl:call-template>
 			</xsl:with-param>
```

Comment:
I still fear that `util-nav-link-attr` won't get called. In understand that the link itself is porject-dependeant, but would it be possible to just output a link, with the call to `util-nav-link-attr` already made ?

Reply:
Oh yeah absolutly. I was suggesting earlier to not make any changes in site nav for this PR and do it in another. But i figure I can still make some minimal adjustments to implement util-nav-link-attr in site-nav.

Reply:
Done!

Reply:
>  I was suggesting earlier

Yeah I did forgot about that sorry ;(